pkgs/development/guile-modules/guile-gcrypt/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, guile, libgcrypt
+, autoreconfHook, pkgconfig, texinfo
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-gcrypt";
+  version = "0.2.1";
+
+  src = fetchurl {
+    url = "https://notabug.org/cwebber/${pname}/archive/v${version}.tar.gz";
+    sha256 = "1qj1yw0kman984x584jjjxnjdhm0pwgp09iyn3b5rqajx7svpqcd";
+  };
+
+  postConfigure = ''
+    sed -i '/moddir\s*=/s%=.*%=''${out}/share/guile/site%' Makefile;
+    sed -i '/godir\s*=/s%=.*%=''${out}/share/guile/ccache%' Makefile;
+  '';
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig texinfo ];
+  buildInputs = [ guile ];
+  propagatedBuildInputs = [ libgcrypt ];
+
+  meta = with stdenv.lib; {
+    description = "Bindings to Libgcrypt for GNU Guile";
+    homepage = "https://notabug.org/cwebber/guile-gcrypt";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ bqv ];
```

Comment:
The commit order doesn't seem right. `bqv` is not defined yet here. The maintainer commit should be the first one.

Reply:
It is actually the first one - presumably you're looking at commit dates or something similar which will be very very wrong by now due to my "creative" rebases